Octokit/Http/RateLimit.cs

```diff
@@ -54,14 +54,14 @@ public RateLimit(int limit, int remaining, long reset)
         /// The date and time at which the current rate limit window resets
         /// </summary>
         [Parameter(Key = "ignoreThisField")]
-        public DateTimeOffset Reset { get; private set; }
+        public DateTimeOffset Reset { get { return ResetAsUtcEpochSeconds.FromUnixTime(); } }
```

Comment:
This was a regression in #860 as `ResetAsUtcEpochSeconds` is the source of truth from the response (I should have called out the `ignoreThisField` earlier). Anyway, the test caught this so yay tests!